test/functional/specs/Personalization/C28756.js

```diff
@@ -32,7 +32,9 @@ test("Test C28756: A form based offer should return if event command contains it
   const alloy = createAlloyProxy();
   await alloy.configure(config);
   const result = await alloy.sendEvent({
```

Comment:
Revert, let's not touch existing tests in this PR.

Reply:
Reverted